src/math.rs

```diff
@@ -0,0 +1,14 @@
+pub fn add(a: i32, b: i32) -> i32 {
+    a + b
+}
+
+#[cfg(test)]
+mod tests {
```

Comment:
If you're writing libs it will do this for you as part of the generated structure of rust-projects in general..